clang/www/analyzer/command-line.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
+          "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+  <title>Running the analyzer from the command line</title>
+  <link type="text/css" rel="stylesheet" href="content.css">
+  <link type="text/css" rel="stylesheet" href="menu.css">
+  <script type="text/javascript" src="scripts/menu.js"></script>
+</head>
+<body>
+
+<div id="page">
+<!--#include virtual="menu.html.incl"-->
+<div id="content">
+
+<h1>Running the analyzer from the command line</h1>
+
+<p>The following tools are supported to run the analyzer from the command line:
+<ul>
+  <li><a href="https://github.com/Ericsson/codechecker">CodeChecker</a> (preferred on Linux)</li>
```

Comment:
Maybe a separate minimal html page would be great for CodeChecker too. Just some basic steps with an example to get a feeling about the usage.
For a more detailed description the user can go to the github page.